### ISSUE TITLE
Java class version 55 for 4.14.2

### ISSUE THREAD

**Issue by ai212983** [jdereg/json-io#200]
`javap -verbose JsonReader | grep major` returns `major version: 55`.
This breaks compatibiltiy JRE 1.8, producing 

```
java.lang.UnsupportedClassVersionError: com/cedarsoftware/util/io/JsonWriter has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0 (unable to load class [com.cedarsoftware.util.io.JsonWriter])
```

Quite a big change for a minor version bump.

upd: checked 4.14.1, it is `major version: 52`, so 4.14.1 is the latest compatible with JRE 1.8.

**Comment by jdereg** [comment]
We are making the changes to get the source back to compliance for 1.8 (semi-regrettfully so).  Lots of new changes going in, and the plan is to maintain 1.8 support and pickup all the new fixed and enhancements, let it stabilize for a while, and then we'll start a 5.0.0 branch that will drop the deprecated code and become JDK11+

**Comment by jdereg** [comment]
We just released json-io 4.15.0.  This is fully JDK 1.8 compatible (class files are version 52).  It has been tested on JDK1.8, JDK11, JDK17, and JDK21.  It was just released a few minutes ago, make take an hour or two to show up on Maven Central.  It is available on Sonatype now.
